enterprise/frontend/src/embedding/data-picker/DataSelector.jsx

```diff
@@ -1068,7 +950,7 @@ const DataSelector = _.compose(
       databases:
         ownProps.databases ||
         Databases.selectors.getList(state, {
-          entityQuery: ownProps.databaseQuery,
+          entityQuery: { saved: true },
```

Comment:
Inline the value here, since we don't want to expose `datbaseQuery` unnecessarily.
https://github.com/metabase/metabase/blob/8fc63096b8d8bfa83d1a4a3878aceac765f0423e/frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/EmbeddingDataPicker.tsx#L108